builtins/util.m4

```diff
@@ -6302,6 +6262,24 @@ done:
 ')
 ')
 
+')
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; packed load and store functions
+;;
+;; These define functions to emulate those nice packed load and packed store
+;; instructions.  For packed store, given a pointer to destination array and 
+;; a varying value, for each lane where the mask is on, the
+;; corresponding value for that lane is stored into packed locations in the
+;; destination array.  For packed load, each lane that has an active mask
+;; loads a sequential value from the array.
+;;
+;; $1: 'TRUE' if intrinsics can be used for '__packed_store_active2'.
```

Comment:
This is correct statement, but would be good to elaborate a little bit. And `intrinsics` -> `LLVM intrinsics`, otherwise it might be confusing.

Reply:
Done